scripts/configs/nginx.conf

```diff
@@ -0,0 +1,36 @@
+#TODO: not working yet... back to apache.
+
+server {
+  server_name hitchwiki.localhost www.hitchwiki.localhost hitchwiki.dev;
+  root /var/nginx/hitchwiki.localhost/www;
+
+  index index.php;
+  client_max_body_size 5m;
+  client_body_timeout 60;
+
+  location ^/en/(profileinfo.php|opensearch_desc.php|api.php|load.php|stylesheets|extensions|resources|skins|config){
+    rewrite ^/(.*)$ /wiki/$1?$args;
+    # root /var/nginx/hitchwiki.localhost/www/wiki/wdwedw;
+  }
+
+  location / {
+    rewrite ^/(.*)$ /wiki/index.php?title=$1&$args;
+  #   try_files /$uri $uri/ @rewrite;
+  }
+
+
+  # location @rewrite {
+  #   rewrite ^/(.*)$ /wiki/index.php?title=$1&$args;
+  # }
+  #
+  # location ^~ /maintenance/ {
+  #   return 403;
+  # }
+  #
+  location ~ \.php$ {
+    fastcgi_pass   unix:/var/run/php-fpm/php-fpm.sock;
+    fastcgi_index  index.php;
+    include        fastcgi.conf;
+    try_files      /$uri $uri/ @rewrite;
+  }
+}
```

Comment:
This should be moved to a separate branch/PR until it works. I like Nginx better too but let's stick to Apache for now? Just to avoid any extra work.